pkgs/development/python-modules/word2vec/default.nix

```diff
@@ -19,12 +20,12 @@ buildPythonPackage rec {
     sha256 = "222d8ffb47f385c43eba45e3f308e605fc9736b2b7137d74979adf1a31e7c8b4";
   };
 
+  nativeBuildInputs = [ setuptools_scm ];
+
   propagatedBuildInputs = [ cython numpy scikitlearn six ];
 
-  checkPhase = ''
-   cd word2vec/tests;
-    ${python.interpreter} test_word2vec.py
-  '';
+  # Checks require test data that needs to be downloaded separately
+  doCheck = false;
```

Comment:
If you're going to disable the tests, could you at least add a `pythonImportsCheck` so we have _some_ hope of detecting breakage?

(also, is there no subset of the tests which _doesn't_ use the test dataset which you could enable?)

Reply:
Thanks I didn't know about `pythonImportCheck`.

The only test that was passing ironically was an import check so it didn't seem worth it. However, I tried downloading the support data to see if I could get that to work and it did. My worry with this version, is that hydra may impose some heavy bandwidth costs on a host that I'm unsure is equiped to handle them. Though, it's obviously better for ensuring the package works.

If this version looks good to you I'll squash the commits and push that up. Otherwise I'll just add the `pythonImportCheck`.

Reply:
How big _is_ the support data?

Reply:
It's around 30 MiB, but the tests only use a small portion of that, notice the head command. The host is a personal website not a company or institution, which is why I hesitate. I expect the package devepers only really expect people to be downloading it infrequently when they setup the project on their dev machines for the first time.

Do you have some idea how often Hydra would end up downloading it? I don't think it maintains a cache for stuff like that.

Maybe I could just check in the test data file? It's only 100KB zipped it's 33KB. I suppose there are licencing implications with that though.

Reply:
I think hydra does keep a cache of these files.

Reply:
And a 33K test file would almost definitely be a no-go. We're even trying to keep patch files out of the repo these days.